core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -160,9 +152,9 @@ object ConsumerGroupCommand {
     }
 
     protected def describeGroup(group: String) {
-      val configs = parseConfigs(opts)
```

Comment:
Even though it's a bit confusing, it seems like it would be better to deprecate it instead of removing it outright. Let's see what @junrao says. If we do remove it, we should maybe fail if the option is present instead of silently ignoring it.